tests/test_cli.py

```diff
@@ -1224,12 +1224,13 @@ def add_compare(self, cat, args, compbase, **kwargs):
 
```

Comment:
To make coverage happy we need to add another test case, this should make it happy:
`c.add_invalid("--machine pc --launchSecurity type=sev-snp,policy=0x24", grep="SEV launch security requires a Q35 UEFI machine", prerun_check="10.5.0")`

Reply:
Added that now

Reply:
I don't know why the coverage report is still complaining about these missing lines when the unit test now covers them.

Reply:
I was trying to figure it as well. We are using fedora:latest container for that job so it has libvirt 10.1.0 that means we will not run these tests so coverage will complain. I did run it locally with newer libvirt and coverage is OK. I'll merge it.